Gruntfile.js

```diff
@@ -37,12 +43,41 @@ module.exports = function (grunt) {
 					dest: 'dist/'
 				}]
 			}
+		},
+		postcss: {
+			options: {
+				map: true,
+				processors: [
+					require('postcss-simple-vars')({
+						variables: require('./src/themes/structural/common')
+					}),
+					require('postcss-modules')({
+						getJSON: function(cssFileName, json) {
+							var filename = path.basename(cssFileName, '.css');
+							fs.writeFileSync(
+								'src/themes/structural/modules/' + filename + '.ts',
+								'export default ' + JSON.stringify(json) + ';'
+							);
+						}
+					})
+				]
+			},
+			dev: {
+				files: [ {
+					expand: true,
+					flatten: true,
+					src: 'src/themes/structural/*.css',
+					ext: '.css',
+					dest: 'src/themes/structural/_generated/'
+				} ]
+			}
 		}
 	});
 
 	grunt.registerTask('dev', [
 		'typings',
-		'tslint',
+		// TODO: uncomment this, figure out a way to ignore generated CSS TS modules
```

Comment:
Added `/* tslint:disable:object-literal-key-quotes quotemark whitespace */` to the beginning of each generated file so we could enable the `tslint` task again.